lib/linux_admin/security/common.rb

```diff
@@ -0,0 +1,17 @@
+module LinuxAdmin
+  class Security
+    module Common
+      def set_value(key, val, filename)
+        config_text = File.read(filename)
+        new_line = "#{key} #{val}\n"
+        new_text = config_text.gsub!(/^#*#{key}.*/, new_line)
+
+        if new_text
+          File.write(filename, new_text)
+        else
+          File.write(filename, new_line, :mode => "a")
+        end
```

Comment:
I was thinking of making this whole idea of set_value a bit more generic with something like:

```ruby
def replace_config_line(new_line, rep_regex, file_text)
  new_text = config_text.gsub!(rep_regex, new_line)
  if new_text
    return new_text
  else
    return new_text << new_line
end
```

Then we can also use this multiple times in the calling class and let it handle saving the file using the returned text.  This way we will always overwrite the contents, but only ever do it once.  What do you think @Fryguy ?